<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObjectMeshAnimationComponentWidgetQt</class>
 <widget class="QWidget" name="GameObjectMeshAnimationComponentWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>304</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh Animation Component</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget, QDialog, QMenuBar::item
{
	background-color: rgb(50, 50, 50);
	color: rgb(255, 255, 255);
}

QHeaderView::section
{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(75, 75, 75, 255));
	padding-left: 4px;
	border: 1px solid rgb(110, 110, 110);
}

QMenuBar::item:selected, QMenu::item:selected
{
    background-color: rgb(120, 120, 120);
	border: 1px solid rgb(200, 200, 200);
}

QPushButton:hover
{
    background-color: rgb(75, 75, 75);
}

QPushButton:pressed
{
    background-color: rgb(100, 100, 100);
}

QLineEdit
{
	border-width: 2px;
	border-style: inset;
	border-color: rgb(60, 60, 60);
	border-radius: 5px;
	padding: 1px;
}

QLineEdit:focus
{
	background-color: rgb(75, 75, 75);
}

QDockWidget::title
{
	background-color: rgb(65, 65, 65);
	padding: 5px;
}

QTabBar::tab 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(80, 80, 80, 255), stop:1 rgba(60, 60, 60, 255));
	padding: 5px;
	borde-width: 3px; 
	border-color: rgb(110, 110, 110);
	border-style: inset;
	border-radius: 8px;
	font-size: 11px;
}

QTabBar::tab:selected 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(120, 120, 120, 255), stop:1 rgba(60, 60, 60, 255));
	border-style: outset;
    border-bottom-color: rgb(60, 60, 60);
}

QTreeWidget
{
	alternate-background-color: rgb(60, 60, 60);
}

QTreeWidget::branch:has-children
{
	image: url(:/AEngine_Editor/Resources/Images/branch-closegrey.png) 0;
}

QTreeWidget::branch:has-children:open
{
	image: url(:/AEngine_Editor/Resources/Images/branch-opengrey.png) 0;
}

QTreeWidget::branch:has-children:open:selected
{
	image: url(:/AEngine_Editor/Resources/Images/branch-openblue.png) 0;
}

QTreeWidget::branch:has-children:!open:selected
{
	image: url(:/AEngine_Editor/Resources/Images/branch-closeblue.png) 0;
}
</string>
  </property>
  <widget class="QTreeWidget" name="m_AnimationsTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>true</bool>
   </property>
   <property name="itemsExpandable">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Animations</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_RemoveAnimationButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Animation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_AddAnimationlButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Animation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PlayTestAnimationButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play Test Animation</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_BlendAnimCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blend Animation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_BlendTimeSB">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>205</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_LoopAnim">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>183</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop Animation</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_BlendAnimTimeLB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Blend Animation Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_SkeletonAssetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesh Skeleton:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_SkeletonAssetTextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>17</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Vertex Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_AddSkeletonAssetButton">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>15</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_ClearSkeletonAssetButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>15</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
